CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+
+## v2.0.0 (WIP)
```

Comment:
There should only be 1 WIP version at a time. Both the features of v1.2.0 and v2.0.0 are unreleased by now, so what you do is that you merge them.

Practically speaking if you only remove the `## v1.2.0 (WIP)` header (and maybe reorganize the notes a little if you feel the need to) and then we only have one set of unreleased features.

Reply:
Do we have a system for designating the breaking change? 
I was thinking something like `Changed (Breaking) ....`

Reply:
We have no convention of that. First and only time I added that was in wharf-helm: https://github.com/iver-wharf/wharf-helm/blob/wharf-helm-1.1.1/charts/wharf-helm/CHANGELOG.md

Where it looks like this:

```md
## v2.0.0 (WIP)

- BREAKING: Added stuff
- BREAKING: Removed other stuff
```